test/unit/extensions/document_snapshot_extension_test.dart

```diff
@@ -5,19 +5,21 @@ import 'package:adventures_in_chat_app/extensions/extensions.dart';
 import '../../mocks/document_snapshot_mocks.dart';
 
 void main() {
-  group('ChatMessage', () {
+  group('DocumentSnapshot', () {
     test(
-        'toMessage will return valid DateTime when valid Timestamp in database',
+        '.toMessage() will return valid DateTime when valid Timestamp in database',
```

Comment:
> in database 

does toMessage return a DateTime? or a Message? 
is a valid Timestamp being in the database what matters? I would imagine there are ways the db could have a timestamp but produce an invalid document snapshot